src/sentry/api/endpoints/organization_user_rollback.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import status
+from rest_framework.permissions import BasePermission
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers.base import serialize
+from sentry.api.serializers.models.userrollback import UserRollbackSerializer
+from sentry.constants import ROLLBACK_ENABLED_DEFAULT
+from sentry.models.organization import Organization
+from sentry.models.rollbackorganization import RollbackOrganization
+from sentry.models.rollbackuser import RollbackUser
+
+
+class MemberPermission(BasePermission):
+    scope_map = {
+        "GET": ["member:read"],
+    }
+
+
+@region_silo_endpoint
+class OrganizationRollbackUserEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+    owner = ApiOwner.ISSUES
+    permission_classes = (MemberPermission,)
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        """
+        Retrieve an organiztion member's rollback
+        """
+        # Validate that the user/org has access to the feature
+        if not features.has("organizations:sentry-rollback-2024", organization, actor=request.user):
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+        # Validate that the organization has rollbacks enabled
+        if not organization.get_option("sentry:rollback_enabled", ROLLBACK_ENABLED_DEFAULT):
+            return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
I think in this case, we want to return some extra detail so that we can tell the user that their org has disabled Rollback

Reply:
Good point, changed this in the most recent commit!